b2share/modules/schemas/root_schemas/bsv/bbmri.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
The root_schema directory should only contain "root schemas" released with B2Share. No community specific schema should be part of the default B2Share.
However specific schemas can be added to the demo just for demonstration purpose.